shaker/libs/pygit2_utils.py

```diff
@@ -1,4 +1,5 @@
 import shaker.libs.logger
+import paramiko
```

Comment:
also true for another library that we have installed already, I forget which one